src/NosCore.Packets/ServerPackets/Arena/BscPacket.cs

```diff
@@ -0,0 +1,21 @@
+﻿//  __  _  __    __   ___ __  ___ ___
+// |  \| |/__\ /' _/ / _//__\| _ \ __|
+// | | ' | \/ |`._`.| \_| \/ | v / _|
+// |_|\__|\__/ |___/ \__/\__/|_|_\___|
+// -----------------------------------
+
+using NosCore.Packets.Attributes;
+using NosCore.Packets.Enumerations;
+
+namespace NosCore.Packets.ServerPackets.Arena
+{
+    [PacketHeader("bsc", Scope.InGame)]
+    public class BscPacket : PacketBase
+    {
+        [PacketIndex(0)]
+        public byte Type { get; set; }

```

Comment:
Sound like en un to me

Reply:
I think too but dunno which value can be take by this packet need to check